exporter/info_test.go

```diff
@@ -17,7 +17,7 @@ func TestKeyspaceStringParser(t *testing.T) {
 	tsts := []struct {
 		db                        string
 		stats                     string
-		keysTotal, keysEx, avgTTL float64
+		keysTotal, keysEx, keysCached, avgTTL float64
```

Comment:
Indentation here seems off. Not sure why the linter didn't catch.

Reply:
fixed